VelocidiSDK/VelocidiSDKTests/VelocidiSDKTests-Bridging-Header.h

```diff
@@ -0,0 +1,3 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
```

Comment:
Yes. Tried to remove it and had some problems when executing the tests in Travis

Reply:
Hum... That seems weird. Can you explain why or paste here the link of the faulty Travis? I'd like to know more.

Reply:
Removing this file in Xcode results in the following error when trying to run the tests (not only in Travis): 

```
2019-05-07 18:35:55.804 xcodebuild[51558:891558]  DVTAssertions: Warning in /Library/Caches/com.apple.xbs/Sources/IDEFrameworks_Fall2018/IDEFrameworks-14460.46/IDEFoundation/Logging/ActivityLog/IDEActivityLogSectionRecorder.m:1050
Details:  log recorder was sent -stopRecordingWithInfo:completionBlock: after it had already been asked to stop recording.
Object:   <IDEActivityLogSectionRecorder: 0x7f8b43f360e0>
Method:   -stopRecordingWithInfo:completionBlock:
Thread:   <NSThread: 0x7f8b4389a900>{number = 15, name = (null)}
Please file a bug at https://bugreport.apple.com with this warning message and any useful information you can provide.

Testing failed:
        Error opening input file '/Users/duarte/Documents/git/velocidi-ios-objc-sdk/VelocidiSDK/VelocidiSDKTests/VelocidiSDKTests-Bridging-Header.h' (No such file or directory)
        Testing cancelled because the build failed.
** TEST FAILED **


The following build commands failed:
        PrecompileSwiftBridgingHeader normal x86_64
        CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(2 failures)
```

Removed this using Xcode so it should remove all references to this file but somewhere in the configurations it is still pointing to it. Even if it is empty, it might be needed because of interoperability and because we have tests in both Objective-C and Swift. But this is just out of the top of my head, I have not researched this, I'll look into it and come back later

Reply:
Yes, that file is needed. The VelocidiSDKTests target is a mixed Objective-C/Swift target and it needs a Bridging header for interoperability. Although we don't have any interoperability between different test classes, we could. More: https://developer.apple.com/documentation/swift/imported_c_and_objective-c_apis/importing_objective-c_into_swift

Reply:
I believe I managed to do it. The build succeeds and the tests pass! Move the file to trash and [follow these steps](https://stackoverflow.com/a/32274718).

Reply:
Nice! Removed it in 9cd4b6e